src/main/java/Hachi.java

```diff
@@ -0,0 +1,266 @@
+import java.util.Scanner;
+
+/**
+ * This program currently starts the chatbot with a greeting,
+ * awaits user input for commands relating to a to-do task manager
+ * and ends off the program with a goodbye message.
+ *
+ * @author clarencepohh
+ * @version 06/02/2024
+ */
+
+public class Hachi {
+
+    /**
+     * Prints a greeting to the user in the console
+     * with the bot's name, Hachi.
+     */
+
+    public static void printGreetingMessage() {
+        String logo = "._. ._.  ._____.  ._____.  ._. ._.  ._.\n"
+                + "| | | |  | ._. |  |  ___|  | | | |  | |\n"
+                + "| |_| |  | |_| |  | |      | |_| |  | |\n"
+                + "| ._. |  | ._. |  | |___   |  _  |  | |\n"
+                + "|_| |_|  |_| |_|  |_____|  |_| |_|  |_|\n";
+
+        System.out.println("Hey, Hachi Here!\n" + logo + "How can I assist you today?\n");
+        spacerInsert("medium", false);
+    }
+
+    /**
+     * Prints to the console a help message that details the commands
+     * the user can use for interacting with the chatbot.
+     *
+     */
+
+    private static void printHelpMessage() {
+        spacerInsert("medium", true);
+        System.out.println("You can use the following commands:");
+        System.out.println("    'list' to retrieve your current list of tasks,");
+        System.out.println("    'mark <#>' to mark task number # as complete,");
+        System.out.println("    'unmark <#>' to mark task number # as incomplete,");
+        System.out.println("    'todo <task name>' to create a to-do,");
+        System.out.println("    'deadline <task name> /by <by date>' to create a task with a deadline,");
+        System.out.println("    'event <task name> /from <start> /to <end>' to create an event with a start and end date,");
+        System.out.println("    'bye' to stop chatting :('");
+        System.out.println("    And if you need to see this again, type 'help'!");
+        spacerInsert("medium", true);
+    }
+
+    /**
+     * Prints to the console a spacer line made of tildes.
+     * Function call has option to choose length of the spacer,
+     * as well as whether there is a 4-space indent before the spacer.
+     *
+     * @param length The desired length of the spacer line. Medium is chosen by default.
+     * @param hasTab Whether the spacer line has a 4-space indent.
+     */
+
+    public static void spacerInsert(String length, boolean hasTab) {
+        String spacer;
+        if (hasTab) {
```

Comment:
please leave a gap between the initialisation of spacer and the if statement, as per the coding standards

Reply:
^^ this!! in the future, separating the two blocks of code allows for easier reading